test/test_ops.py

```diff
@@ -479,11 +494,19 @@ def test_python_ref_torch_fallback(self, device, dtype, op):
         self._ref_test_helper(contextlib.nullcontext, device, dtype, op)
 
     @unittest.skipIf(TEST_WITH_ASAN, "Skipped under ASAN")
-    @onlyCUDA
     @ops(python_ref_db)
     @parametrize('executor', ['aten', 'nvfuser'])
     @skipIfTorchInductor("Takes too long for inductor")
     def test_python_ref_executor(self, device, dtype, op, executor):
+        if device == "mps" and executor == 'nvfuser':
+            return
+        MPS_BLOCKLIST = [
```

Comment:
@skotapati , instead of the BlockLIst can you create xfaillist ? If it's failing, we should add the expected failure list. If there is a crash, then we indeed need to fix it.